geoip/.gitignore

```diff
@@ -0,0 +1,3 @@
+GeoIP.conf
```

Comment:
Since we now have an empty file in `geoip` directory, can we move these definitions to the `.gitignore` at the root level?

Reply:
Done in 1364eaf79cab9150e64b91422434a9bf9a0e39be.